### DISCUSSION TITLE
Confirm to open link externally in browser

### DISCUSSION THREAD

**Discussion by MicrocontrollersDev** [ppy/osu#30740] [Ideas] [answered]
Since buying supporter, I commonly find myself accidentally clicking the supporter icon next to my name when opening my player card from the top header bar. Instead of taking me to my profile page in game, it takes me out of the game to my browser to show the osu supporter page. It would be nice if this button had a confirmation dialogue asking "are you sure you want to open externally?" or something.

While I believe a confirmation dialogue might be good for other instances of opening external links (ie main menu cards), this is the only one that annoys me, so I would be fine with a different approach as well.

**Comment by peppy** [comment]
I'm not sure this should link in the first place from your own card, but regardless we can probably add confirmation locally to this flow.

**Reply by vatei** [reply]
Now that I think about it there's no way to get supporter in game is there?

**Comment by peppy** [comment]
@cl8n @arflyte might be something to consider with user cards – careful placement of clickable elements to avoid users inadvertently clicking them and ending up in an unexpected place when actually trying to open the user's profile.

**Reply by arflyte** [reply]
Noted.

**Comment by MicrocontrollersDev** [comment, marked as answer]
https://github.com/ppy/osu/pull/31442
